### PR TITLE
Regenerate the alerts

### PR DESCRIPTION
**Pull request by psiinon**
Many of the file names have changed, and some new alerts have been added as well.
This also picks up some terminology changes.

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

File: site/content/docs/alerts/90034.md [hunk 1]
```diff
@@ -24,8 +24,12 @@ Do not trust any user data in NGINX configs. In this case it is probably the use
 
 * https://www.nginx.com/blog/trust-no-one-perils-of-trusting-user-input/
 
+### CWE: [0](https://cwe.mitre.org/data/definitions/0.html)
+
+### WASC:  0
+
 ### Code
 
  * [org/zaproxy/zap/extension/ascanrulesAlpha/CloudMetadataScanner.java](https://github.com/zaproxy/zap-extensions/blob/master/addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/CloudMetadataScanner.java)
```
Comment (kingthorin) [inline site/content/docs/alerts/90034.md:33]: Looks like some of the changes have been missed.

Was this assembled manually or automatically?

Comment (psiinon) [reply site/content/docs/alerts/90034.md:33]: Looks like the script wasnt updated correctly .. fixing that now...

**Comment by psiinon** [comment]
Hopefully all fixed now...

**Comment by kingthorin** [comment]
Something is still wrong, most of the content appears to be being deleted.

**Comment by psiinon** [comment]
Ugh, yeah :( Will look at it asap...

**Comment by psiinon** [comment]
OK, _now_ I think its ok...

**Pull request merged**